packages/backend/migrations/20240524102509_drop_quote_ratio_columns.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.alterTable('quotes', (table) => {
+    table.decimal('minExchangeRateNumerator', 64, 0).alter()
+    table.decimal('minExchangeRateDenominator', 64, 0).alter()
+    table.decimal('lowEstimatedExchangeRateNumerator', 64, 0).alter()
+    table.decimal('lowEstimatedExchangeRateDenominator', 64, 0).alter()
+    table.decimal('highEstimatedExchangeRateNumerator', 64, 0).alter()
+    table.decimal('highEstimatedExchangeRateDenominator', 64, 0).alter()
```

Comment:
needed something bigger than 20, (I don't think we will ever need more than 20 ever), but just being generous so we don't run into this issue again

Reply:
Seems reasonable. Inspecting the db shows these are stored as `numeric` types (same as decimal) in postgres. Found this in the docs about how much storage this would take:

> Numeric values are physically stored without any extra leading or trailing zeroes. Thus, the declared precision and scale of a column are maximums, not fixed allocations. (In this sense the numeric type is more akin to varchar(n) than to char(n).) The actual storage requirement is two bytes for each group of four decimal digits, plus three to eight bytes overhead.

https://www.postgresql.org/docs/current/datatype-numeric.html

So it varies on actual size of the data, not just this declaration. So I don't really see a downside of leaving lots of extra headroom.

Reply:
Were you able to test this migration against any data? Since we're going from lower to higher precision I imagine it probably works fine but figure it's worth testing.

Reply:
@BlairCurrey I just made some transactions on `main` then rebuilt the backend and continued making transactions without issue